manifests/vhost.pp

```diff
@@ -2368,7 +2368,7 @@
   concat::fragment { "${name}-serversignature":
     target  => "${priority_real}${filename}.conf",
     order   => 90,
-    content => template('apache/vhost/_serversignature.erb'),
+    content => epp('apache/vhost/_serversignature.epp'),
```

Comment:
The file has no substitution, does it make sense to use a template rather than a file?

Reply:
we tried removing but it was failing the acceptance so created the file

Reply:
I think @smortex suggested to use `file()` instead of `epp()`. Then store it in `files/` instead of `templates`.

Reply:
sure will try that